src/components/pages/add-review/add-review-form.test.js

```diff
@@ -41,11 +41,11 @@ describe(`Test AddReviewForm`, () => {
     );
 
     expect(screen.getByText(`Post`)).toBeInTheDocument();
-    userEvent.type(screen.getByPlaceholderText(`Review text`), `test comment`);
+    userEvent.type(screen.getByTestId(`add-review_textarea`), `test comment`);
     expect(screen.getByDisplayValue(/test comment/i)).toBeInTheDocument();
   });
 
-  it(`Click on SubmitBtn works`, () => {
+  it(`is submitted when Submit btn is clicked`, () => {
```

Comment:
старайся всегда писать констркуцию `it should...`

```suggestion
  it(`should submit when Submit button is clicked`, () => {
```